compiler/testData/codegen/boxInline/simple/inlineCallInInlineLambda.kt

```diff
@@ -0,0 +1,12 @@
+// FILE: 1.kt
+
```

Comment:
package test

Reply:
Please add package directive and corresponding import in second section

Reply:
Done